src/50_Samples_%26_Templates/Product_Page.html

```diff
@@ -647,7 +647,9 @@ <h4 class="p2">Products that have been bought together</h4>
     </div>
     <!--
     With AMP, you can easily pull in different latest offers or highlights without changing the page. To do so, just use `amp-list` to fire a CORS request to a JSON endpoint which supplies the list of related products. These are populated into an amp-mustache template on the client.
-     Learn more about `amp-list` [here](/components/amp-list/)
+    You can dinamically change the source of the `amp-list` CORS JSON endpoint by using `amp-bind`: we are using a variable called `relatedProductsSrc` and we are binding its value
```

Comment:
suggestion, rephrase to "we bind the variable `relatedProductsSrc` to the to the `src` attribute of the `amp-list`"